docs/about/release_notes.rst

```diff
@@ -32,9 +32,14 @@ Version 6.0
 
 Breaking changes:
 * Use parameters from storage instead of ensemble_config (`#5674 <https://github.com/equinor/ert/pull/5674>´_)
+* Remove ANALYSIS_COPY (`#5826 <https://github.com/equinor/ert/pull/5826>´_)
+* Pass in storage to BatchSimulator.start (`#5656 <https://github.com/equinor/ert/pull/5656>´_)
```

Comment:
Is this something we actually  need to communicate?

Reply:
Maybe not very relevant, but is a breaking change to external API